server/chat-plugins/poll.ts

```diff
@@ -363,10 +370,8 @@ export const commands: ChatCommands = {
 				this.modlog('CLEARQUEUE');
 				this.sendReply(this.tr`Cleared poll queue.`);
 			} else {
-				const [slotString, roomid, update] = target.split(',');
 				const slot = parseInt(slotString);
 				const curRoom = roomid ? (Rooms.search(roomid) as ChatRoom | GameRoom) : room;
-				if (!curRoom) return this.errorReply(this.tr`Room "${roomid}" not found.`);
```

Comment:
Why was this deleted? This will cause a crash if it doesn't exist, won't it?

Reply:
we really shouldn't be using `curRoom` at all, `room` is updated to the new room above.

Reply:
You approved this... :/